### ISSUE TITLE
New failure in mn-samples-nist: text(...)()

### ISSUE THREAD

**Issue by opoudjis** [plurimath/plurimath#276]
In the latest release done today, a formula that was parsing correctly until now has just stop parsing:

````
Plurimath::Math.parse('text(Output_Entropy)()', :asciimath)
````

succeeds. But

````
Plurimath::Math.parse('text(Output_Entropy)()', :asciimath).to_mathml
````

crashes. It was previously generating the following output:

````
<math>
<mstyle displaystyle="false">
    <mtext>Output_Entropy</mtext>
    <mrow>
      <mo>(</mo>
      <mo>)</mo>
    </mrow>
  </mstyle>
</math>
````

Other formulas aborting in https://github.com/metanorma/metanorma-cli/actions/runs/10036536375/job/27739398612, mn-samples-nist, are:

````
X = text(next)()
text(next)()
A = text(next)()
````

**Comment by opoudjis** [comment]
A similar failure is happening in mn-samples-jcgm:

````
Plurimath::Math.parse('"Pr"[]', :asciimath).to_mathml
````

**Comment by opoudjis** [comment]
This is now holding up Metanorma release.

**Comment by opoudjis** [comment]
Will let you know when I have compiled locally all of mn-samples-jcgm and mn-samples-nist with the fixed branch locally.

**Comment by opoudjis** [comment]
NIST finished, JCGM still running.

**Comment by suleman-uzair** [comment]
@opoudjis just released version `0.8.12`.

**Comment by opoudjis** [comment]
Thank you!
